sample/DotNetIsolated/Blob/BlobTriggerBindingSamples.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+
+namespace SampleApp
+{
+    public static class BlobTriggerBindingSamples
```

Comment:
Are these sample functions covering different scenarios from the ones in BlobInputBindingSamples.cs?

Reply:
Yup here we're testing the BlobTrigger binding, the other ones are testing the input binding